attention_and_transformers.ipynb

```diff
@@ -0,0 +1,4395 @@
+{
```

Comment:
<p>Suggestion (optional): I would move the self attention to before this section and ask them to implement a self attention mechanism using the previous functions. Template to be completed would be something like:</p><p> </p><p> </p><p><code>def self_attention(inputs, mask=None):</code></p><p><code>	return attention, attention_weights</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>